.github/workflows/test-build.yml

```
@@ -6,7 +6,7 @@ jobs:
      runs-on: [ubuntu-latest]
      strategy:
        matrix:
-         go_version: [1.13]
+         go_version: [1.18]
```

why not 1.19 ?

however I'd test it against mutliple versions

```suggestion
         go_version: [1.18, 1.19]
```

---

upgraded to 1.19